deflect/qt/QmlStreamer.cpp

```diff
@@ -66,6 +66,11 @@ QQmlEngine* QmlStreamer::getQmlEngine()
     return _impl->getQmlEngine();
 }
 
+Stream *deflect::qt::QmlStreamer::getStream()
```

Comment:
As a matter of fact, we DO have a REST interface...!
Maybe we should start using that instead of the deflect sendCommand() which is marked private in doxygen...